onnxruntime/test/contrib_ops/quantize_ops_test.cc

```diff
@@ -207,13 +226,13 @@ TEST(QuantizeLinearContribOpTest, QuantizeLinear_per_tensor_float_int8) {
                                   });
   test.AddInput<float>("y_scale", {}, {2.0f});
   test.AddInput<int8_t>("y_zero_point", {}, {1});
-  test.AddOutput<int8_t>("y", dims, {1, 2,
-                                     3, -1,
-                                     2, 0,
-                                     3, -1,
-                                     127, -127,
-                                     127, -127,
-                                     127, -128,
+  test.AddOutput<int8_t>("y", dims, {1, 2,       // Add comments to keep these aligned
+                                     3, -1,      //
+                                     2, 0,       //
+                                     3, -1,      //
+                                     127, -127,  //
+                                     127, -127,  //
+                                     127, -128,  //
                                      127, -128});
```

Comment:
nit: start the initializer list on a new line and clang-format will leave the line breaks alone